L/libsingular_julia/build_tarballs.jl

```diff
@@ -9,7 +9,7 @@ uuid = Base.UUID("a83860b7-747b-57cf-bf1f-3e79990d037f")
 delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
 
 name = "libsingular_julia"
-version = v"0.47.2"
+version = v"0.48.0"
```

Comment:
Why not
```suggestion
version = v"0.47.3"
```
?

Reply:
> This was done the last time when you guys updated the lobcxxwrap minor version.
And I think we want to enable Pkg using this new build together with bumping the CxxWrap.jl compat in Polymake.jl.
I have no idea what happens if it would try to resolve the old CxxWrap.jl 0.16.x with this jll build against a "newer" libcxxwrap.
>
> But you have done this for a longer time, so please just apply the patch of you are sure it doesn't break anything

My comment from libpolymake_julia. Just replace all Polymake by singular.